src/patternfly/components/Tabs/tabs.scss

```diff
@@ -733,3 +779,35 @@ $pf-v6-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl
   }
 }
 // stylelint-enable
+
+@media (prefers-reduced-motion: no-preference) {
+  .#{$tabs}:not(.pf-m-box) {
+    .#{$tabs}__link,
+    .#{$tabs}__item.pf-m-action {
+      &::after {
+        content: revert;
+      }
+    }
+
+    .#{$tabs}__list::after {
+      position: absolute;
+      inset-block-start: var(--#{$tabs}--link-accent--InsetBlockStart, var(--#{$tabs}--link-accent--start));
+      inset-block-end: var(--#{$tabs}--link-accent--InsetBlockEnd);
+      inset-inline-start: var(--#{$tabs}--link-accent--InsetInlineStart, var(--#{$tabs}--link-accent--start));
+      width: var(--#{$tabs}--link-accent--Width, var(--#{$tabs}--link-accent--length));
+      height: var(--#{$tabs}--link-accent--Height, var(--#{$tabs}--link-accent--length));
+      content: "";
+      border: 0 solid var(--#{$tabs}--link-accent--color);
+      border-block-end-width: var(--#{$tabs}--link-accent--BorderBlockEndWidth);
+      border-inline-start-width: var(--#{$tabs}--link-accent--BorderInlineStartWidth);
+      transition-timing-function: var(--#{$tabs}--link-accent--TransitionTimingFunction);
+      transition-duration: var(--#{$tabs}--link-accent--TransitionDuration);
+      transition-property: --#{$tabs}--link-accent--length, --#{$tabs}--link-accent--start, width;
+    }
+  }
+
+  .#{$tabs}.pf-m-initializing-accent {
+    --#{$tabs}--link-accent--TransitionDuration--length: 0;
+    --#{$tabs}--link-accent--TransitionDuration--start: 0;
```

Comment:
Should these two lines now just be `--#{$tabs}--link-accent--TransitionDuration: 0;`?